_common

```diff
@@ -144,10 +144,17 @@ handle_unknown() {
             group_name=$(echo "$group_data" | runjq -r '.[0].name')
             job_name=$(echo "$job_data" | runjq -r '.job.name')
             job_result=$(echo "$job_data" | runjq -r '.job.result')
-            job_info="* Name: $job_name
+            info="* Name: $job_name
 * Result: $job_result
-* Reason: $reason"
-            email="[$job_name]($testurl)"$'\n'"$header"$'\n'"$job_info"$'\n'$'\n'"$excerpt"
+* Reason: $reason
+
+It might be a product bug, an outdated needle, test code needing adaption or a
```

Comment:
```suggestion
It might be a product bug, an outdated needle, test code needing adaptation or a
```

Although the words are similar, adaption is something else. An adaption that dogs have compared to wolves is being able to process more starches found in human foods.

Reply:
Looks like you're right (https://grammarist.com/usage/adaption-adaptation). Thanks for pointing things like this out (only that way one can learn).